erikiado/jp2_online
# Task/recuperar estudios capturista

#### What's this PR do?
Adds the functionality to recover deleted studies.

#### Where should the reviewer start?
estudios_socioeconomicos/forms.py to check the actual form
estudios_socioeconomicos/test_forms.py
captura/views.py
captura/tests.py

also check the static files.
#### How should this be manually tested?
run the tests.
Also delete a study, check that it appears in the tab of Recuperar estudio, and recover it, checking that it appears in the listing of studies.

#### Any background context you want to provide?
there's a lot of files changed due to some refactoring.


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
